src/data_format_conversions.jl

```diff
@@ -107,7 +125,30 @@ function _convert_data!(
         end
         if component["__metadata__"]["type"] ∈ ["BatteryEMS", "GenericBattery"]
             component["__metadata__"]["type"] = "EnergyReservoirStorage"
-            component["storage_technology_type"] = "StorageTech.OTHER_CHEM"
+            component["storage_technology_type"] = IS.serialize(StorageTech.OTHER_CHEM)
+            soc_min = component["state_of_charge_limits"]["min"]
+            soc_max = component["state_of_charge_limits"]["max"]
+            if soc_max == 0.0
+                component["storage_capacity"] = 0.0
+                component["storage_level_limits"] = Dict("min" => 0.0, "max" => 1.0)
+                (component["initial_energy"] != 0.0) && throw(
+                    ArgumentError(
+                        "Maximum state of charge is zero but initial energy is not; cannot parse",
+                    ),
+                )
+                component["initial_storage_capacity_level"] = 0.0
+            else
+                # Derive storage_capacity from old state of charge limits and normalize new
+                # state of charge limits, initial capacity accordingly
```

Comment:
This is going to be dependent on the data that the user had and we have no way of knowing the prior assumptions. We should just throw a warning that we are doing the conversion.

Reply:
Added a warning.